staff/anna-nolla/blog-layout/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>blog-skylab</title>
+    <link rel="stylesheet" href= "style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Skylab Code Academy</h1>
+        <h2>Blog exercise</h2>
+    </header>
+
+    <nav>
+        <ul id="menu">
+            <li>
+                <a href="/">Home</a>
+            </li>
+            <li>
+                <a href="">Articles</a>
+            </li>
+            <li>
+                <a href="">Coments</a>
+            </li>
+            <li>
+                <a href="">Contact</a>
+            </li>
+        </ul>
+    </nav>
+
+    <main>
+        <section>
+           <h1>How to Find And Use Long Tail Keywords to Drive More Search Traffic Even If You’re A Beginner</h1>
+           <p> Hands down, search traffic is the best traffic source anyone can use to build a profitable blog that makes money. But there’s one issue with getting more traffic from search engines: it usually takes a lot of time (especially if you’ve a new website). Here’s where long tail SEO comes into play. Long tail keywords send you the most qualified visitors from search engines like Google and they are really easier to rank for no matter what industry you are in.</p>
+            <br>
```

Comment:
It'd be better if you didn't use `<br>` to create white space. You should work with margin or padding. In this case you could add `margin-bottom` to your `<p>` element.

Reply:
changes done